sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -12833,5 +12833,12 @@ ALTER TABLE userroles_history ADD COLUMN restrictAccessToAssignedEntities boolea
 
 INSERT INTO schema_version (version_number, comment) VALUES (536, 'Assign case(s) to a User and allow them to see the data of only the assigned case(s) in the system #12697');
 
+-- 2023-12-18 Move hide jurisdiction fields feature property to dedicated feature type #12806
+INSERT INTO featureconfiguration (id, uuid, creationdate, changedate, enabled, featuretype)
+VALUES (nextval('entity_seq'), generate_base32_uuid(), now(), now(), (SELECT properties::jsonb->'HIDE_JURISDICTION_FIELDS' FROM featureconfiguration WHERE featuretype = 'CASE_SURVEILANCE')::boolean, 'HIDE_JURISDICTION_FIELDS');
```

Comment:
For the users which have PostgreSQL below 14 ( 10, 11, 12) this conversion from jsonb to boolean will not work, an error will be thrown and the deployment will fail.

I have PostgreSQL 10 ( I did not work in the past on the Postgres version update ticket and I did not update yet) and @adinaflorea9  has PostgreSQL 12. We tested on both postgres versions and the same error is thrown:

![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/9d615cf4-4066-40ff-8758-32d093e0542b)
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/776ff5d4-669c-43fe-8806-941886ca0313)

@MateStrysewske The question is if should we keep this conversion and everybody who has PostgreSQL below 14 will need to update the version in order to have a running development ( do not know if this makes sense since the project is ending soon) or to change this line to a conversion compatible with older Postgres versions too?  :)